templates/starter/bicep/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure Developer CLI (azd) Bicep Starter
+
+A starter template with [Bicep](https://aka.ms/bicep) as infrastructure provider for [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview) (azd).
+
+The following assets have been provided:
+
+- Infrastructure-as-code (IaC) files under the `infra` folder that demonstrate how to provision resources and setup resource tagging for `azd`.
+- A [dev container](https://containers.dev) configuration file under the `.devcontainer` directory that installs python, Node.js, and infrastructure tools by default. This can be readily used to create cloud-hosted developer environments such as [GitHub Codespaces](https://aka.ms/codespaces).
+- Continuous deployment workflows for CI providers such as GitHub Actions under the `.github` directory, and Azure Pipelines under the `.azdo` directory that work for most use-cases.
```

Comment:
IMHO, I would also provide an azd-friendly .gitignore and mention the git repository we create on init.

I would also add a section below with `## next steps` to list what should a user try to do next... like create applications or how to connect source code with infra...

Reply:
I'll try to have this match something close to our ToDo templates.

Reply:
Let's have the discussion [here](https://github.com/Azure/azure-dev/pull/2055#discussion_r1182875850)

Reply:
.gitignore is already provided: https://github.com/Azure-Samples/azd-starter-bicep/blob/pr/2055/.gitignore